pkg/pillar/cmd/zedagent/handlevolume.go

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2017-2018 Zededa, Inc.
```

Comment:
Please updated copyright year to 2017-2020

Reply:
Will do.

Reply:
Resolved.